scripts/prepare_sector_network.py

```diff
@@ -1630,7 +1649,9 @@ def insert_electricity_distribution_grid(
     - Micro-CHP units
     """
 
-    nodes = n.buses.query("carrier == 'AC' and not index.str.contains('DRES')").index
+    nodes = n.buses.query(
+        "carrier == 'AC' and not index.str.contains('DRES') and not index.str.contains('OH')"
+    ).index
```

Comment:
That is a very nitty gritty intervention, any idea how we can make the filter here more robust which works for upstream in general?

Reply:
For upstream I introduced this PR [here](https://github.com/PyPSA/pypsa-eur/pull/1672/files) which basically takes into account all the AC buses instead of the pop_layout.index for creating the lv buses

Reply:
What about using the `type` field? This field is already populated for Offshore Hubs and could easily be populated for `DRES` as well. This would simplify the expression to:

```suggestion
    nodes = n.buses.query("carrier == 'AC' and type==''").index
```